File: Checker source code with regex generated/Factorial check/test/tc4/tes4.cpp
#include <cstdio>
#include <cstdlib>
#include <iostream>

/*
 * fungsi faktorial
 * param int n
 * return n!
 */
int fac(int n){
	int d = 1;
	for (int i=2; i<=n; i++){
		d *= i;
	}
	return d;
}

int main(){
	int a,b;
	// baca nilai a dan b
	scanf("%d%d",&a,&b);//comment wkwk
	// tulis nilai a + b
	printf("%d\n",a+b);
	int n = a;
	int d = fac(n);
	printf("%d %d\n",a,d);
	return 0;
}

File: Checker source code with regex generated/Factorial check/test/tc4/tes4.pas
var a,b,d,i,j:integer;

(*
 * fungsi faktorial
 * param int n
 * return n!
 *)
function fac(n:integer):integer;
begin
	j := 1;
	for i:=2 to n do
	begin
		j := j*i;
	end;
	fac := j;
end;

begin
	// baca nilai a dan b
	readln(a,b); //comment wkwk
	// tulis nilai a + b
	writeln(a+b);
	{
		comment lagi
	}
	d := fac(a);
	writeln(a,' ',d);
end.
